source/_static/css/mattermost-global.css

```diff
@@ -1999,3 +1999,10 @@ mark {
 .mermaid text#task20-text {
     fill: white !important;
 }
+
+/* NESTED TABS */
+#check-your-mattermost-client-settings .tab-set .tab-set {
+    background-color: var(--color-background-secondary);
+    padding: 1rem 1rem;
+    margin-top: -2rem;
+}
```

Comment:
Based on the latest content structure, this needs to be changed slightly, here's the updated version:
```
/* NESTED TABS */
#check-your-mattermost-client-settings .tab-set .tab-set {
    background-color: var(--color-background-secondary);
    padding: 0rem 1rem 1rem 1rem;
}
```